src/sst/elements/merlin/Makefile.am

```diff
@@ -50,6 +50,11 @@ libmerlin_la_SOURCES = \
 	topology/singlerouter.cc \
 	topology/hyperx.h \
 	topology/hyperx.cc \
+	topology/polarfly.cc \
+	topology/polarfly.h \
+	topology/polarstar.cc \
+	topology/polarstar.h \
+	topology/polar.H \
```

Comment:
I don't see a polar.H file in the changed files.  Either this is here on accident or you're missing a file.  Also, if you're missing the file, please make sure it is polar.h and not polar.H.